app/models/micropost.rb

```diff
@@ -1,9 +1,18 @@
 class Micropost < ApplicationRecord
   validates :content, presence: true, length: { minimum: 5};
   belongs_to :user
+  belongs_to :category
   has_rich_text :content
   has_many_attached :images
 
+  def self.category(category)
+    if category != nil and category != ""
+      self.where('category_id = :category', category: category)
```

Comment:
```ruby
self.where(category_id:, category)
```

Reply:
oh, I don't know that syntax, thank you very much :D